old/bin/aa.pl

```diff
@@ -792,8 +792,8 @@ sub form_header {
       <th align=left>$form->{invtotal}</th>
      <td>| . (($form->{currency} ne $form->{defaultcurrency})
               ? $form->format_amount(\%myconfig,
-                                     $form->{invtotal}
-                                     * $form->{exchangerate}, 2) : '') . qq|</td>
+                                     $form->parse_amount( \%myconfig, $form->{invtotal} )
+                                     * $form->parse_amount( \%myconfig, $form->{exchangerate} ), LedgerSMB::Setting->new(%$form)->get('decimal_places')) : '') . qq|</td>
```

Comment:
I'm not sure we should want to do more than what you're doing (that is: parsing the exchange rate), but the same function first formats the exchangerate here: https://github.com/ledgersmb/LedgerSMB/blob/3008375e4e764ff2acc7c2a2b564fa41e6844a05/old/bin/aa.pl#L425

Reply:
Assigning a formatted_amount to the `$form->{exchangerate}` will truncate precision for a value which is used in 10 other modules